src/pudl/transform/eia930.py

```diff
@@ -0,0 +1,224 @@
+"""Module to perform data cleaning functions on EIA930 data tables."""
+
+import pandas as pd
+from dagster import AssetOut, Output, multi_asset
+
+import pudl
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+@multi_asset(
+    outs={
+        "core_eia930__hourly_balancing_authority_net_generation": AssetOut(
+            io_manager_key="parquet_io_manager"
+        ),
+        "core_eia930__hourly_balancing_authority_demand": AssetOut(
+            io_manager_key="parquet_io_manager"
+        ),
+    },
+    compute_kind="pandas",
+)
+def core_eia930__hourly_balancing_authority_assets(
+    raw_eia930__balance: pd.DataFrame,
+):
+    """Transforms raw_eia923__balance dataframe.
+
+    Extract the net generation information from the balance table and extract fuel type
+    from the net generation columns.
+    """
+    qual_cols = [
+        "report_datetime_local",
+        "report_datetime_utc",
+        "balancing_authority_code_eia",
+        # report_date
+        # eia_region_code,
```

Comment:
if we are not loosing it entirely (i.e. we can merge it back in later in `out_`) i don't think they should be here

Reply:
Right now we're doing different things in different tables.  If we do want it to be in the table we're presenting to the public, I think we should probably just leave it in here for now since we aren't making any separate output tables (and also having both core and output tables for tens of millions of rows is going to be a lot of not very helpful bloat)